lib/modules/hideSidebar.js

```diff
@@ -0,0 +1,116 @@
+/* @flow */
+import { $ } from '../vendor';
+import { Module } from '../core/module';
+import { i18n } from '../environment';
+import * as Menu from './menu';
+import {
+	CreateElement
+} from '../utils';
+
+export const module: Module<*> = new Module('hideSidebar');
+
+module.moduleName = 'hideSidebarName';
+module.category = 'productivityCategory';
+module.description = 'hideSidebarTabDesc';
+
+let hideSidebar;
+let propToChange;
+let valueToChangeTo;
+let valueToChangeFrom;
+let $side;
+let $content;
+let contentOffset;
+let headerHeight;
+let documentHalf;
+let sideContent;
+let sideWidth;
+let rulesCheck = ['reddiquette', 'rules'];
+
+const toggleSidebar = () => {
+
+	if (!$content.data('sidebar-hidden')) {
+		let check = true;
+		if (rulesCheck.some(function (v) { return sideContent.indexOf(v) >= 0; })) {
+			check = confirm("Have you read the rules?");
+		}
+		if (check) {
+			doHideSidebar();
+		}
+	} else {
+		doShowSidebar();
+	}
+}
+
+function recursiveEach($element) {
+	$element.children().each(function () {
+		let $this = $(this);
+		let thisOffset = $this.offset();
+		if ($this.css("position") == "absolute" &&
+			thisOffset.top >= headerHeight &&
+			thisOffset.top <= contentOffset.top &&
+			thisOffset.left < documentHalf
+		) {
+			$this.css("visibility", "visible");
+		}
+		recursiveEach($this);
+	});
+
+}
+
+const doHideSidebar = () => {
```

Comment:
These could simply be named `hideSidebar` and `showSidebar`.

Reply:
Done.